lumigator/infra/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,125 @@
+name: lumi
```

Comment:
Why are we creating a duplicate docker-compose instead of using the one in .devcontainers?

Reply:
+1 to this, if there is not too much difference and as you are parametrizing this, perhaps we can have one single compose file and two different envs?